src/dao/DAO.js

```diff
@@ -44,42 +26,43 @@ class DAO<TEntity: { id: EntityID }, TEntityMutator> extends BaseDAO<
 
     this.__resolveSingle(
       this.__buildHandler().find(this.__reformatIDValue(id)),
-      null,
-      'delete'
+      /* params */ {},
+      'delete',
     )
       .then(() => {
         this._entityLoaderByID.delete(id);
         this._clearQueryCaches();
         this._emitChanges();
       })
-      .catch((error) => this._updateCacheForError(id, error));
+      .catch((error: Error): void => this._updateCacheForError(id, error));
   }
 
   count(queryOptions?: QueryOptions): LoadObject<number> {
```

Comment:
Can you test this or have you tested it?

Reply:
ok, fixed

Reply:
Does the query actually return entities? We should figure out a way to just return the count if it does.

Reply:
Response from the server looks like {data:[], inlinecount: 123}, data is blank array